torch/fx/experimental/symbolic_shapes.py

```diff
@@ -4411,7 +4411,11 @@ def _maybe_evaluate_static(
                 # Skip var_ranges logic for SingletonInt which is only used
                 # for jagged layout NestedTensors today
                 continue
-            vr = var_ranges[k]
+            try:
+                vr = var_ranges[k]
+            except KeyError:
+                log.warning("%s is not in var_ranges, defaulting to unknown range.", k)
```

Comment:
May I know why it is a warning here? Would it always warn on unbacked symbols? The cpp template uses some unbacked symbols to build IR nodes for codegen which would always get warning here.
https://github.com/pytorch/pytorch/blob/695502ca653d25be23be61a909587f110067e859/torch/_inductor/codegen/cpp_gemm_template.py#L110